src/components/form/Form.js

```diff
@@ -90,7 +90,17 @@ export default class FormComponent extends Component {
     if (!this.formSrc && this.options.formio) {
       const rootSrc = this.options.formio.formsUrl;
       if (this.component.form && isMongoId(this.component.form)) {
-        this.formSrc = `${rootSrc}/${this.component.form}`;
+        if (
```

Comment:
This change seems very specific and worries me that this is a band-aid for a problem that would be fixed elsewhere. Let's talk through this in Dev Support.